shared/util/codeql/util/test/InlineExpectationsTest.qll

```diff
@@ -645,6 +645,29 @@ module TestPostProcessing {
     private import InlineExpectationsTest as InlineExpectationsTest
     private import InlineExpectationsTest::Make<Input>
 
+    bindingset[loc]
+    private predicate parseLocation(
```

Comment:
I would like to avoid having to parse locations; see https://github.com/asgerf/codeql/pull/2 for an alternative.